tools/topology2/sof-cnl-nocodec.conf

```diff
@@ -0,0 +1,94 @@
+
+#
+# Simple Machine - High level topology - Maps to machine driver.
+#
+#
+# PCM 0 <-> bufferN.0 <-> volumeN.0 <-> bufferN.1 -> SSP0
+#
+
+<include/common/tokens.conf>
+<include/platform/intel/bxt.conf>
+<include/pipelines/pipeline-volume-playback.conf>
+<include/pipelines/pipeline-volume-capture.conf>
+<include/common/connection.conf>
+<include/common/pipe_endpoint.conf>
+<include/components/dai.conf>
+<include/common/data.conf>
+<include/dais/ssp.conf>
+<include/common/manifest.conf>
+
+#
+# Pipeline definitions
+#
+
+# Pipeline ID:1 PCM ID: 0
+Object.volume-playback."1" {
+	pipeline_id	1
+	pcm_id		0
+	pcm_name	"Port0"
+	format		"s24le"
+
+	# set the mixer name for PGA.0
+	pga.0.mixer.0.name "1 Master Playback Volume"
```

Comment:
this is interesting. Looks good! Wondering what other implications or potential uses this can have. Alternatively we could have just `primary_mixer_name` or something similar and just propagate it. Would probably be easier for the compiler, but less elegant.

Reply:
I started of with passing with a primary_mixer_name/primary_switch_name but there could  be cases where there are more than 1 pga widget in a pipeline. THen we'll have to pass 4 names. This way we know exactly what we're setting